### DISCUSSION TITLE
DaVinci Resolve export gets "The clip was not found" when source file is from a Sony camera

### DISCUSSION THREAD

**Discussion by mckayc** [WyattBlue/auto-editor#380] [4. Bugs]
### Bug description

If I run auto-editor with a Sony (Sony a6500 camera specifically) mp4 file, the resulting XML file when importing to DaVinci Resolve results in "The clip was not found" error. If I first add the clip to the media pool, right click on the clip, go to clip properties and set the Timecode to 00:00:00:00 then I can import just fine. I have attached a gif so you can see the exact issue. 

![2023-08-13 -- 22 40 17 - Resolve_25jufzZbG6](https://github.com/WyattBlue/auto-editor/assets/3774563/736e98c9-a96b-4b5b-9cdd-46dae2462002)

### What's your platform?

```shell
Platform: Windows 10 amd64
FFmpeg Version: N-110322-g0c25da5a28-g1440bf15e2+1
FFmpeg Path: C:\Python311\Lib\site-packages\ae_ffmpeg\Windows\ffmpeg.exe
Auto-Editor Version: 23w32a
```

### What properties does your input video have?

 - video:
   - track 0:
     - codec: h264
     - fps: 60000/1001
     - resolution: 1920x1080
     - aspect ratio: 16:9
     - pixel aspect ratio: 1:1
     - duration: 18.518500
     - pix fmt: yuv420p
     - color range: tv
     - color space: bt709
     - color primaries: bt709
     - color transfer: iec61966-2-4
     - timebase: 1/60000
     - bitrate: 50353767
     - lang: und
 - audio:
   - track 0:
     - codec: pcm_s16be
     - samplerate: 48000
     - channels: 2
     - duration: 18.518500
     - bitrate: 1536000
     - lang: und
 - container:
   - duration: 18.5185
   - bitrate: 52549781

### Comments

I feel like this is specific to files from Sony cameras. If I transcode the mp4 file to another mp4 file and run auto-editor and import to DaVinci Resolve everything works as expected. So this may not in fact be a bug but something odd with Sony files. I totally get that this is pretty specific and may not even be a bug but thought I would list it here with the workaround.

Just want to say thanks for creating auto-editor. I use this multiple times a day and it is a lifesaver! I have actually tried a handful of commercial software similar to this and prefer this over those options. Because this is CLI I was able to write a Powershell script to automate my preferred settings and I can literally open and  run this app on my video files in less than 5 seconds. Keep up the great work!

**Comment by WyattBlue** [comment]
I'll look into it this Thursday.

**Comment by WyattBlue** [comment]
I think switching the fcp7 backend to the fcp11 should solve this indirectly as well as #368. 

Try installing from master to see if this fixes the issue
```
pip uninstall auto-editor
pip install git+https://github.com/WyattBlue/auto-editor.git
```

**Comment by mckayc** [comment]
Thanks. I just updated to 23w33a-dev (23.33.1) but I am still experiencing the same issue.

Changing the time on the clip attribute still functions as a workaround.


![2023-08-21 -- 06 08 36 - Resolve_P9EZwrKOqp](https://github.com/WyattBlue/auto-editor/assets/3774563/5e8d9649-b012-4f68-a4dd-0386306f7a68)

![2023-08-21 -- 06 11 27 - Resolve_K1XuMB2AHN](https://github.com/WyattBlue/auto-editor/assets/3774563/8bff3dd6-9cd5-4c99-a648-a771eeb1ba1a)

**Reply by vitharana** [reply]
Thanks a lot, I used your trick to get the thing working

**Comment by Tendaliu** [comment]
If the Timecode of the media doesn't start at 00:00:00:00 this error would occur

**Comment by SuperNova74-K** [comment]
Hello @mckayc , @Tendaliu @WyattBlue ,
I re-solved :) this issue by editing the video's path in the XML file by changing the back-slashes to forward slashes

For example the file path `<pathurl>` for me was: 

`<pathurl>C:\Users\walid\Videos\Departments\IS\TheVid.m4v</pathurl>`

And I was getting the error so I switched it to : 

`<pathurl>C:/Users/walid/Videos/Departments/IS/TheVid.m4v</pathurl>`

And it worked like a charm so you might want to edit this in the output.py as I'm currently a bit too busy to do it myself, if you don't have time I will try to do it next Saturday.

Try it and tell me if it worked for you :)

**Reply by btwiamamin** [reply]
Yes it worked! Thanks man.

**Comment by virtualflo** [comment]
I have the same problem here with a video recorded with the new Blackmagic Camera app. My pathurl got the correct forward slashes and I first add the clip to the media pool, right click on the clip, go to clip properties and set the Timecode to 00:00:00:00 then I can import just fine like @mckayc

**Reply by virtualflo** [reply]
Just to add a point, when I'm trying to update the time code of the clip, then the video and the audio isn't synchro. Don't know why.

**Comment by WyattBlue** [comment]
This is fixed in new versions. Auto-Editor now applies the proper startpoint when exporting for DaVinci Resolve.
